src/cli/commands/ls.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+const {print, rightpad} = require('../utils')
```

Comment:
We are not using destructuring just yet. Could you change it?

Why? We review the JS best practices every quarter or so and commit to a set of things, this helps us agree on codestyle and move forward faster :)